app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: :comments_counter
+end
```

Comment:
- [ ] Comment model should have a method that updates the comments counter for a post so would you add this method as well?